### DISCUSSION TITLE
Not pairing

### DISCUSSION THREAD

**Discussion by Ryandiabetic** [NightscoutFoundation/xDrip#2234] [Q&A]
Hey I'm new to xdrip and I can't get my phone to request to pair with my dexcom g6 transmitter. I have the newest xdrip version and a pixel 6 phone. When I first got my dexcom a couple of weeks ago the dexcom app connected perfectly but I have now uninstalled that to get xdrip to work. Can anybody help me?

**Comment by Navid200** [comment]
Please post a screenshot of the system status page.  You can blackout the transmitter ID if you like.

**Reply by Ryandiabetic** [reply]
No worries thanks
![Screenshot_20220722-112026~2](https://user-images.githubusercontent.com/109741945/180355799-8607a4b1-9170-4acb-a591-ed6287cfb7e0.png)

**Reply by Navid200** [reply]
As a first step, please go over all settings and ensure they all match the following:
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

**Reply by Navid200** [reply]
If you have a sensor that was functioning in the Dexcom app, you don't need to start again in xDrip.
After you establish connectivity with the transmitter, if you have a session in progress, it will just start showing readings as explained here:  https://navid200.github.io/xDrip/docs/Session-in-progress.html

**Reply by Ryandiabetic** [reply]
My settings match the ones you have attached except I don't have the Special pairing workaround option on my settings at all?

**Reply by Ryandiabetic** [reply]
I have already ended my dexcom session but could restart if you think it would help

**Reply by Navid200** [reply]
No, you should not touch stop or start sensor for now.

You should update your xDrip version to the following:  https://github.com/NightscoutFoundation/xDrip/releases/tag/2022.06.09
You just need to install over your existing version.

There is no way your settings match the ones I have shown.
Looking at your system status page screenshot, you can see you have preemptive restart enabled.  That setting being enabled is not a problem.  However, it is not a recommended setting.  Therefore, your settings are not identical.

Please look at every single setting on every one of the different pages included.

**Reply by Ryandiabetic** [reply]
No problems I will recheck now thank you I will also get the version you have attached

**Reply by Ryandiabetic** [reply]
Have now re checked all settings and updated the app but I still don't have that special pairing workaround option

**Reply by Navid200** [reply]
Don't worry about that one.  That setting appears first in the July 13 release.  But, it's there to help with Android 11.  
You are using Android 12, right?

Are you sure you don't have a typo in the transmitter ID?  It is normal to mistake a 0 with an O.

**Reply by Ryandiabetic** [reply]
I can redo but I'm sure it's correct as it worked for dexcom. Yea I am using Android 12

**Reply by Navid200** [reply]
Are you sure you have uninstalled Dexcom app?  They cannot both communicate with the same transmitter at the same time.

**Reply by Ryandiabetic** [reply]
Yea I uninstalled before I installed xdrip to try and save the conflict

**Reply by Navid200** [reply]
OK,

Let's try something else.  Do the following.

1- On the system status page, swipe right to go to the classic status page.  Tap on forget device.
2- Go to Settings -> Hardware Data Source and change the last character of the transmitter ID to something else and apply.
3- Correct the last character and apply.
4- go back to the system status page.

**Reply by Ryandiabetic** [reply]
Have done that now thanks again for all your help

**Comment by Ryandiabetic** [comment]
Is there an actual apply button for settings? As I'm just pressing ok. Is that the same thing

**Reply by Navid200** [reply]
There is an apply when you change the transmitter ID.
When you, for example, check "Authenticate G5 before each read", that's it.  As soon as you check or uncheck, the setting changes.  But, it's a good idea to restart the phone after you have corrected all the settings just to be sure.

**Reply by Ryandiabetic** [reply]
No problems. It doesn't feel like it's even trying to find the transmitter that's all. In the events log it keeps saying Bluetooth audio connected to Galaxy watch could that be a problem?

**Reply by Navid200** [reply]
It could only be a problem if you have a watch that can directly act as a collector from the G6 transmitter.  In that case, it would have interfered with the Dexcom app as well.

**Reply by Ryandiabetic** [reply]
I just turned off watch collection and my phone has now asked to pair and is connected. Thanks so much for all your help really appreciate it

**Reply by Ryandiabetic** [reply]
Is there anything else that needs to be done now it's paired?

**Reply by Navid200** [reply]
No problem.
I am curious to know more about the watch.  
It would be great if you don't mind providing the brand and model of the watch.

This is how you should start a sensor:
https://navid200.github.io/xDrip/docs/Starting-G6.html

**Reply by Ryandiabetic** [reply]
It's a Samsung Galaxy watch 4 classic
